.github/workflows/lint.yml

```diff
@@ -0,0 +1,38 @@
+name: Lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: purcell/setup-emacs@master
+        with:
+          version: snapshot
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.6"
+          architecture: "x64"
+
+      - uses: conao3/setup-cask@master
+        with:
+          version: 0.8.4
+
+      - name: Resolve dependency
+        run: cask install
+
+      - name: Run Indentation Lint
+        run: |
+          git diff-tree --no-commit-id --name-only -r ${{ github.sha }} \
+          | grep \.el | grep -v \.dir-locals\.el \
+          | xargs -t -I^ cask emacs -q --batch ^ --eval "\
+              (let ((require-final-newline t)) \
+                (require 'flycheck) \
+                (eval-buffer) \
+                (untabify (point-min) (point-max)) \
+                (indent-region (point-min) (point-max)) \
+                (save-buffer))" \
```

Comment:
I think we could move that to our Makefile, then we could lint locally, and CI just call the makefile like the other CI workflows, WDYT?

Reply:
I don't get why it needs to be specified on cask files, I thought of just moving that piece of code to makefile and calling it from here, just like other makefile tasks we already do that

Reply:
because current shell script depends `${{ github.sha }}` placeholder.

Reply:
Hum, got it
Ok, we can keep it here for now I think